Version 5.3.0b3: ModuleNotFoundError: No module named 'jwt'

```python
pip install --pre redis
python -c "import redis"
```

```python
    import redis
/usr/local/lib/python3.12/site-packages/redis/__init__.py:3: in <module>
    from redis import asyncio  # noqa
/usr/local/lib/python3.12/site-packages/redis/asyncio/__init__.py:1: in <module>
    from redis.asyncio.client import Redis, StrictRedis
/usr/local/lib/python3.12/site-packages/redis/asyncio/client.py:35: in <module>
    from redis.asyncio.connection import (
/usr/local/lib/python3.12/site-packages/redis/asyncio/connection.py:30: in <module>
    from ..auth.token import TokenInterface
/usr/local/lib/python3.12/site-packages/redis/auth/token.py:4: in <module>
    import jwt
E   ModuleNotFoundError: No module named 'jwt'
```

Intoduced in https://github.com/redis/redis-py/pull/3445.

@vladvildanov Could you have a look?

--- a/dev_requirements.txt
+++ b/dev_requirements.txt
@@ -16,4 +16,4 @@ uvloop
 vulture>=2.3.0
 wheel>=0.30.0
 numpy>=1.24.0
-redis-entraid @ git+https://github.com/redis-developer/redis-py-entraid.git@main
+redis-entraid==0.1.0b1

--- a/dev_requirements.txt
+++ b/dev_requirements.txt
@@ -16,4 +16,4 @@ uvloop
 vulture>=2.3.0
 wheel>=0.30.0
 numpy>=1.24.0
-redispy-entraid-credentials @ git+https://github.com/redis-developer/redispy-entra-credentials.git/@main
+redis-entraid @ git+https://github.com/redis-developer/redis-py-entraid.git@main

--- a/setup.py
+++ b/setup.py
@@ -38,6 +38,7 @@ setup(
     python_requires=">=3.8",
     install_requires=[
         'async-timeout>=4.0.3; python_full_version<"3.11.3"',
+        "PyJWT~=2.9.0",
     ],
     classifiers=[
         "Development Status :: 5 - Production/Stable",

